src/oc_erchef/apps/oc_chef_wm/src/oc_chef_wm_base.erl

```diff
@@ -84,9 +84,11 @@ service_available(Req, #base_state{reqid_header_name = HeaderName} = State) ->
 
 %% @doc Validate the requested X-Ops-Server-API-Version value and populate base_state,
 %% and reply with it as or reply with error if it's not valid.
+%%
+%% If X-Ops-Server-API-Version is not sent, the server assumes an API version of 0.
 -spec server_api_version(undefined|string()) -> api_version() | {error| string()}.
 server_api_version(undefined) ->
-    ?API_MIN_VER;
+    0;
```

Comment:
Note: this is biggest functional change. We will default to zero if no header passed!